features/step_definitions/web_steps.rb

```diff
@@ -0,0 +1,59 @@
+Given(/^I visit the homepage$/) do
+  visit('/')
+end
+
+When(/^I follow "(.*?)"$/) do |link|
+  click_link(link)
+end
+
+When(/^I click button "(.*?)"$/) do |button|
+  click_button(button)
+end
+
+Then(/^I see a "(.*?)" message$/) do |message|
+  expect(page).to have_content(message)
+end
+
+
+Then(/^I see a blank board$/) do
+  expect(page).to have_content('0')
+end
+
+Given(/^I have a single square game$/) do
+  step('I visit the homepage')
+  step('I click button "New Game"')
+end
+
+When(/^I get the computer to make a move$/) do
+  fill_in('number_moves', :with => '1')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.